File: cli/cmd/logout.go
package cmd

import (
	"fmt"

	"github.com/alperen-albayrak/kutup/cli/internal/session"
	"github.com/spf13/cobra"
)

var logoutCmd = &cobra.Command{
	Use:   "logout",
	Short: "Clear stored session",
	RunE:  runLogout,
}

func runLogout(cmd *cobra.Command, args []string) error {
	store, err := session.Open(profile)
	if err != nil {
		return err
	}
	defer store.Close()
	if err := store.ClearSession(); err != nil {
		return err
	}
	fmt.Println("Logged out.")
	return nil
}

File: crates/kutup-cli/src/commands/logout.rs
//! `kutup logout` — mirrors `cmd/logout.go`.

use anyhow::Result;

use crate::session::Store;

pub fn run(profile: &str, json: bool) -> Result<()> {
    let store = Store::open(profile)?;
    store.clear_session()?;
    if json {
        crate::output::print_json(&serde_json::json!({
            "loggedOut": true,
            "profile": profile,
        }))?;
    } else {
        println!("Logged out.");
    }
    Ok(())
}
